xtask/src/main.rs

```diff
@@ -69,6 +84,7 @@ fn install_tools() -> Result<()> {
         vec!["mdbook-pandoc", "--locked", "--version", "0.9.3"],
         vec!["mdbook-i18n-helpers", "--locked", "--version", "0.3.5"],
         vec!["i18n-report", "--locked", "--version", "0.2.0"],
+        vec!["mdbook-linkcheck", "--locked", "--version", "0.7.7"],
```

Comment:
There is a github action that installs this manually [here](https://github.com/google/comprehensive-rust/blob/dbd1a09031714bda7be972ea14686f44d8cbe451/.github/workflows/build.yml#L166). Can that action be updated to instead run `cargo xtask install-tools` which will then ensure linkcheck is installed? I'm concerned with having two places where we define this version, but don't have much context about that github action.

Reply:
I have reverted installing linkcheck as part of install-tools because it results in build failures of some languages in the CI.